Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -24,8 +56,10 @@ ES_PORT='9200'
 
 CURL_N_MAX=10
 SLEEP=5
-DELIMITER=`echo -e -n "\x00"`
-EOProcess=`echo -e -n "\x06"`
+EOBatch=$"\4"
```

Comment:
Why use `$"\4"`?
[Dollar sign + single-quoted string](https://www.gnu.org/software/bash/manual/html_node/ANSI_002dC-Quoting.html) (`$'some string'`) makes escape sequences in the string to be interpreted (that\`s why you have used it in Stage 60 to place code `4` into a variable: `DELIMITER=$'\4'`).
[Dollar sign + double-quoted string](https://www.gnu.org/software/bash/manual/html_node/Locale-Translation.html) (`$"some string"`) will cause the string to be translated according to the current locale. Is that what you mean?

Seems like pure `"\4"` would give exactly the result we need here, and save a reader from looking for some hidden  meaning in it ;)

Reply:
As you use `echo -ne "$EOBatch"` later, it was not fatal ;) Yet with `$'\4'` it will work, too.